packages/block-library/src/tag-cloud/edit.js

```diff
@@ -105,6 +105,15 @@ function TagCloudEdit( { attributes, setAttributes, taxonomies } ) {
 		setAttributes( updateObj );
 	};
 
+	const serverSideRenderAttirbutes = {
+		...attributes,
+		className: undefined,
+		style: {
+			...attributes.style,
+			spacing: undefined,
```

Comment:
If border support is added in the future, it will be necessary to write `border: undefined` here.

Reply:
There is an open PR [adding typography supports to the Tag Cloud](https://github.com/WordPress/gutenberg/pull/43452) block. This would need updating here, in that PR, or immediately after it lands. 

This is probably further justification that we should move the logic for stripping block support related attributes for `<ServerSideRender>` as [suggested](https://github.com/WordPress/gutenberg/pull/44438#pullrequestreview-1119477064) over on the PR doing this for the Archive block.